### DISCUSSION TITLE
Mailing list or discussions

### DISCUSSION THREAD

**Discussion by nyue** [waltligon/orangefs#90] [Q&A] [answered]
Should questions and discussion about orangefs take place here or on the mailing list ?

I visited http://www.orangefs.org/#home-section, the link to the mailing list returns a 404 error.

Cheers

**Comment by boydwilson** [comment, marked as answer]
We opted to use a hosting company for OrangeFS discussions and we have had not great results, so we have just been discussing the same question.  We are active on the Linux kernel discussions related to storage subsystems, so you can see those types of things there (the 5.13 kernel page cache integration is working nicely FWIW).  But for normal help and assistance, we did discuss just using GitHub discussions.  We are open to input, but for now, yes posting messages here would probably be the most effective discussion method/tool.  We will get the website updated to reflect this for now.
